src/views/clusteroverview/MonitoringTab/migrations-card/components/MigrationsTable/components/MigrationPolicyTooltip/MigrationPolicyTooltip.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useMemo } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { Tooltip } from '@patternfly/react-core';
+
+import { MigrationTableDataLayout } from '../../utils/utils';
+
+import { migrationsConfigTooltipMap } from './utils';
+
+type TooltipConfigObject = { label: string; value: string };
+
+type MigrationPolicyTooltipProps = {
+  obj: MigrationTableDataLayout;
+  children?: React.ReactElement<any, any>;
+};
+
+const MigrationPolicyTooltip: React.FC<MigrationPolicyTooltipProps> = ({ obj, children }) => {
+  const { t } = useKubevirtTranslation();
+
+  const migrationConfigurations = useMemo(() => {
+    const mpConfig: TooltipConfigObject[] = [];
+    const defaultConfig: TooltipConfigObject[] = [];
+    const pushConfigField = (
+      field: string,
+      getDisplayText: (value: string | boolean | number) => string,
+      label: string,
+    ) => {
+      if (obj?.mpObj?.spec?.[field] !== undefined)
+        return mpConfig.push({ label, value: getDisplayText(obj?.mpObj?.spec?.[field]) });
+      if (obj?.migrationsDefaultConfigurations?.[field] !== undefined) {
+        return defaultConfig.push({
+          label,
+          value: getDisplayText(obj?.migrationsDefaultConfigurations?.[field]),
+        });
+      }
+      return undefined;
+    };
+    migrationsConfigTooltipMap.forEach(({ field, getDisplayText, label }) => {
+      pushConfigField(field, getDisplayText, label);
+    });
+
+    return {
+      defaultConfig,
+      mpConfig,
+    };
+  }, [obj]);
+
+  return (
+    <Tooltip
+      content={
+        <>
+          {!isEmpty(migrationConfigurations?.mpConfig) && (
+            <>
+              <div>
+                <b>{t('MigrationPolicy values')}</b>
+              </div>
+              {migrationConfigurations?.mpConfig.map(({ label, value }) => (
+                <div key={value}>
+                  {label}: {value}
+                </div>
+              ))}
+            </>
+          )}
+          {!isEmpty(migrationConfigurations?.defaultConfig) && (
+            <>
+              <div>
+                <b>{t('Default values')}</b>
+              </div>
+              {migrationConfigurations?.defaultConfig.map(({ label, value }) => (
+                <div key={value}>
+                  {label}: {value}
+                </div>
+              ))}
+            </>
+          )}
+        </>
+      }
+    >
+      {children}
```

Comment:
why pass the children and not do the rendering here?

Reply:
I wanted some flexibility with the rendered children in case I'll need to use this tooltip in another place.
since it's not needed at the moment I can insert the children inside the component and if needed in the future will change back, thanks :)